.github/workflows/discord_updater.yml

```diff
@@ -0,0 +1,41 @@
+name: Discord Doc Updater
+
+on:
+  schedule:
+  - cron: "0 15 * * 3"
```

Comment:
For now, Github Actions will run the script, create a branch, push it, and create a PR (I prefer to keep a human in the loop for the final merge) every Wednesday (once per week is probably enough).